fecgov/fec-eregs
# bump lodash version

### Summary
Replacement of dependabot PR #515 

### How to test
@rfultz: my solution was to manually add lodash as a devDependency and npm i. Can you focus your portion of the review on the correctness of this approach (I assumed just adding to package-lock.json would mean anyone deleting and reinstalling to create a new package-lock could inadvertently roll back the lodash version)

@pkfec @fec-jli: either or both of you could review. 
- check out branch
- `snyk test --file=package.json` and `snyk test --file=package-lock.json`
- run local ergs and try parsing (both worked for me and the local api formatting was fine)
